src/main/java/cinema/model/User.java

```diff
@@ -17,6 +19,8 @@
     private String email;
     @Column(nullable = false)
     private String password;
+    @ManyToMany
+    private List<Role> roles;
```

Comment:
better to use Set here since roles should not repeat

Reply:
Fixed